lib/nightmare.js

```diff
@@ -91,6 +91,22 @@ function Nightmare(options) {
     debug('electron child process exited with code ' + code + ': ' + help[code]);
   });
 
+  Object.defineProperty(this, 'engineVersions', {
+    get: function() {
+      delete this.engineVersions; 
+      return this.engineVersions = JSON.parse(
```

Comment:
If I’m getting this right, you’re intending to wipe out the getter function and replace it with a static value. Is that right? If so, this doesn’t work—you can only `delete` if `configurable` is `true` in the property descriptor (it defaults to false) and you can only re-set the value with the `=` operator if `writable` is true (also defaults to false). The `=` operator still returns the value you tried to set even if it didn’t work, so the failure isn’t always obvious (one of those really not-so-great design decisions in JS).